### PR TITLE
#8 create: sidebar + header

### PR DESCRIPTION
**Pull request by kennytruong06**

### CODE REVIEW

File: components/layout/Header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            </div>
+            <div className="w-[50px] h-[100%] flex justify-start items-center">
+              <div className="w-[25px] h-[25px] flex justify-center items-start  overflow-hidden">
+                <BsFillBoxSeamFill color="#000000" fontSize={"25px"}  />
+              </div>
+            </div>
+          </Link>
+        </SectionLogin>
+
+        {/* <SectionProfile>
```
Comment (lamxuanthinh) [inline components/layout/Header/index.tsx:63]: you should clear the code command

File: Views/Profile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                        <ImgAvatarProfile src={dataUser.pathAvatar} alt="avatar-profile" />
+                </SectionAvatar>
+                </SectionCoverImage>
+                <SectionDescription>
+                        <NameUser>{dataUser.nameUser}</NameUser>
+                        <DescriptionUser>
+                            {dataUser.description}
+                        </DescriptionUser>
+                        <BtnEditProfile>
+                            <i className="fa-solid fa-pen icon-edit-profile"></i>
```
Comment (lamxuanthinh) [inline Views/Profile/index.tsx:39]: You can handle the style `icon `in the file  `ProfileStyled.ts` ...

File: components/layout/Header/headerStyled.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+export const HeaderInput = styled.input`
+    width: 100%;
+    height: 40px;
+    background-color: transparent;
+    border: none;
+    outline: none;
+
+
```
Comment (lamxuanthinh) [inline components/layout/Header/headerStyled.ts]: Please code clean

File: pages/index.tsx [hunk 4]
```diff
@@ -1,7 +1,14 @@
-import type { NextPage } from "next";
```
Comment (lamxuanthinh) [inline pages/index.tsx:1]: You should use ` { NextPage }`

**Review by lamxuanthinh — CHANGES_REQUESTED** [review]

File: components/layout/Header/index.tsx (hunk 1 above)
Comment (kennytruong06) [reply components/layout/Header/index.tsx:63]: This is the code prepare after login, I don't know how to handle if I deletes it, and after that how to use it again when I want

File: Views/Profile/index.tsx (hunk 2 above)
Comment (kennytruong06) [reply Views/Profile/index.tsx:39]: this is the code from LMD

**Comment by lamxuanthinh** [comment]
Good

**Pull request merged**